locales/en-US/app.ftl

```diff
@@ -228,7 +228,7 @@ hidePastExperiments = Hide Past Experiments
 
 ## Text of warnings to the user if various error conditions are detected
 warningGenericTitle = Something is wrong!
-warningGenericDetail = Something has gone wrong with Test Pilot. Please <a>file a bug</a> and mention this error message.
+warningGoToFAQDetail = Something has gone wrong with Test Pilot. Please <a>check the FAQs</a> to learn more.
```

Comment:
I'm not sure that "to learn more" is the best choice of words. You should check the FAQ to find possible explanations for the error, or solutions.

Anyway, that's an English problem, l10n wise it's good.